sagemaker-clarify/text_explainability/text_explainability.ipynb

```diff
@@ -7,20 +7,30 @@
     "# Explaining text sentiment analysis using SageMaker Clarify"
```

Comment:
<p>I would add types to the function definition</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3793/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='64'/>